### PR TITLE
Add sal generics page

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: python/lsst/ts/xml/generate_sal_generics_doc.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_xml
+#
+# Developed for the LSST Data Management System.
```
Comment (wvreeven) [inline python/lsst/ts/xml/generate_sal_generics_doc.py:3]: Please change this line to `Developed for the Vera Rubin Observatory Telescope and Site Systems.`.

File: python/lsst/ts/xml/generate_sal_generics_doc.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
```
Comment (wvreeven) [inline python/lsst/ts/xml/generate_sal_generics_doc.py:20]: Please add this line `# along with this program.  If not, see <https://www.gnu.org/licenses/>.` at the bottom of the header.

**Review by wvreeven — APPROVED** [review]
Looks good to me. Please take my two comments into account.

**Comment by patrickingraham** [comment]
Can you please add a one or two sentence introduction at the top of the page?
Just saying what generics are, that they apply to all CSCs, and that they are further explained in LSE-209.

**Pull request merged**